webview/src/shared/components/sectionContainer/SectionContainer.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { MouseEvent } from 'react'
+import { Section as PlotsSection } from 'dvc/src/plots/webview/contract'
+import { MessageFromWebviewType } from 'dvc/src/webview/contract'
+import styles from './styles.module.scss'
+import { Icon } from '../Icon'
+import { ChevronDown, ChevronRight, Info } from '../icons'
+import { isSelecting } from '../../../util/strings'
+import { isTooltip } from '../../../util/helpers'
+import { sendMessage } from '../../vscode'
+import Tooltip from '../tooltip/Tooltip'
+import { IconMenu } from '../iconMenu/IconMenu'
+import { IconMenuItemProps } from '../iconMenu/IconMenuItem'
+
+export const SectionDescription = {
```

Comment:
[F] This needs to be here to avoid circular imports and to make the types available to `SectionDecription[section]`. I think that having all of the info sections in the same place will be beneficial for style as well. LMK if you disagree.